src/models/platform.ts

```diff
@@ -87,6 +89,16 @@ export class PlatformInformation {
         return getRuntimeDisplayName(this.runtimeId);
     }
 
+    public isMacVersionLessThan(version: string): boolean {
+        if (this.isMacOS) {
+            let versionInfo = plist.parse(fs.readFileSync('/System/Library/CoreServices/SystemVersion.plist', 'utf-8'));
```

Comment:
>plist.parse(fs.readFileSync('/System/Library/CoreServices/SystemVersion.plist', 'utf-8')) [](start = 30, length = 89)

not sure, but this looks like the type of code that could throw...it way be good to try/catch this method to avoid any surprises